scarb/tests/e2e/subcommand.rs

```diff
@@ -76,6 +77,40 @@ fn env_variables_are_passed() {
         .success();
 }
 
+#[test]
+#[cfg(unix)]
+fn env_scarb_log_is_passed_verbatim() {
+    let t = TempDir::new().unwrap();
+    write_script(
+        "env",
+        indoc! {
+            r#"
+            #!/bin/bash
```

Comment:
This is more portable as far as I know, not every OS keeps bash in `/bin` (I think NixOS could be one example)

```suggestion
            #!/usr/bin/env bash
```